packages/components/src/components/hds/copy/snippet/index.hbs

```diff
@@ -6,6 +6,7 @@
   type="button"
   class={{this.classNames}}
   {{hds-clipboard text=@textToCopy onSuccess=this.onSuccess onError=this.onError}}
+  aria-label={{concat 'copy ' @textToCopy}}
```

Comment:
a comment in the [initial request](https://hashicorp.slack.com/archives/C7KTUHNUS/p1709145109837559) was:

>  There is no announcement that the text was copied to clipboard so I wonder if someone who isn't sighted might have the perspective that nothing happens when they click on the button?

should we consider also this?

Reply:
I think we should discuss it because it is a lot trickier to do. I would make a separate issue to track the conversation and potential implementations. But I don't think it should block this more immediate bugfix. WDYT?

Reply:
works for me 👍